api/api-reply.js

```diff
@@ -28,12 +41,40 @@ router.post('/reply', function(request, response) {
 	});
 });
 
+router.post('/uploadMedia', function(request, response) {
+	var user = request.body.user || '',
+		creds = request.body.creds || {},
+		message = request.body.message || '',
+		access = request.body.access || {},
+		filename = request.body.filename || '',
+		_ = request.body._ || !1,
+		path = request.body.path || '',
+		params = {
+			media: path + filename
+		},
+		twitter = _ ? TwitterClient._Client(creds) : TwitterClient.Client(creds);
+	twitter.uploadMedia(params, access.accessToken, access.accessTokenSecret, function(err, data, res) {
+		if (err) {
+			return response.status(err.code)
+				.send(err.message);
```

Comment:
@balthazzar error struct contains no err.code record, example from the server logs
```
{ statusCode: 401,
  data: '{"errors":[{"code":89,"message":"Invalid or expired token."}]}' }
```
when executed with wrong parameters this code again will crash server. Please pay attention to error handling.
Also we have constant /api/search request, i quoted above. Please check tokens on the server and if needed update them.